web-common/src/features/dashboards/selectors/index.ts

```diff
@@ -73,7 +73,6 @@ export const getFilterSearchList = (
           measureNames: [metricsExplorer.leaderboardMeasureName],
           timeStart: timeControls.timeStart,
           timeEnd: timeControls.timeEnd,
-          limit: "15",
```

Comment:
This was removed because of the design changes to the dropdown. Previously, the drop down only displayed the already selected values plus a search bar that would bring up more. The new dropdown ostensibly shows the full list of available options and even provides an opportunity to select or deselect all.

Reply:
Hmmm would probably be better to have an explicit limit of 100. Otherwise a frontend engineer will need to look at go code to know the limit.